product_code_builder/views/product_attribute_value_view.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0"?>
-<odoo>
-
-    <record id="variants_tree_view" model="ir.ui.view">
-        <field name="name">product.attribute.view.tree (inherited in product_code_builder)</field>
-        <field name="model">product.attribute.value</field>
-        <field name="inherit_id" ref="product.variants_tree_view"/>
```

Comment:
`"product.variants_tree_view"` doesn't exist anymore so I delete this file. Please let me know if this file should exist.